code/modules/admin/topic.dm

```diff
@@ -2630,7 +2630,7 @@
 		fax_panel(usr)
 
 	else if(href_list["getplaytimewindow"])
-		if(!check_rights(R_ADMIN))
+		if(!check_rights(R_ADMIN | R_MOD | R_MENTOR))
```

Comment:
Wouldnt really call this a fix. This panel is used by admins too and also shows job unlocks.

Reply:
Yeah that's fair. I wasn't sure if fix or tweak would be more appropriate here since it seems (at least to me) like it was intended to be accessible to mentors in the first place.

Reply:
Frankly I think everyone should be able to view their own playtime stats.
And it only shows jobs unlocked via playtime, I believe.